_content-types/documents.md

```diff
@@ -14,10 +14,10 @@ If there is a user need for a PDF, [make the PDF accessible](/accessibility-incl
 
 Make Word documents accessible to everyone even if you are emailing them internally. This includes people who rely on assistive technologies, such as screen readers.
 
-Microsoft has guidance on [making Word documents more accessible ](https://support.office.com/en-us/article/Creating-accessible-Word-documents-d9bf3683-87ac-47ea-b91a-78dcacb3c66d?CTT=3&CorrelationId=b3c40249-a7fa-4300-9c12-5d0f79242a78&ui=en-US&rs=en-US&ad=US).
+Microsoft has guidance on <a href="https://support.office.com/en-us/article/Creating-accessible-Word-documents-d9bf3683-87ac-47ea-b91a-78dcacb3c66d?CTT=3&amp;CorrelationId=b3c40249-a7fa-4300-9c12-5d0f79242a78&amp;ui=en-US&amp;rs=en-US&amp;ad=US" rel="external">making Word documents more accessible </a>.
```

Comment:
It's kramdown specific: https://kramdown.gettalong.org/quickref.html#block-attributes